docs/guides/upgrading-v3.md

```diff
@@ -0,0 +1,263 @@
+
+When the `serverless` CLI is installed globally and locally (in the project’s `node_modules`), the local version will always be used. It is no longer possible to disable that behavior ([learn more](../deprecations.md#support-for-enablelocalinstallationfallback-setting-is-to-be-removed)).
+
+Finally, the `serverless studio` command has been removed: that feature was deprecated and is no longer available.
+
+### Service configuration
+
+The default Lambda runtime has changed from NodeJS 12 to NodeJS 14, given this is now the default runtime recommended by AWS.
+
+In `serverless.yml`, the `service` key no longer accepts a YAML object ([learn more](../deprecations.md#service-property-object-notation)).
+
+```yaml
+# Deprecated in v2, removed in v3:
+service:
+    name: my-service
+
+# Correct syntax:
+service: my-service
+```
+
+All options that used to be defined inside the `service` key have been moved to other sections (mentioned below in that document). This change clears up confusion that existed between the `service` and `provider` sections.
+
+### API Gateway
+
+When configuring API Gateway, some configuration options have moved to a dedicated sub-section of `provider`. That will help clear up confusion with similar `httpApi` settings.
+
+```yaml
+provider:
+  # Deprecated in v2, removed in v3:
+  apiKeys: ...
+  resourcePolicy: ...
+  usagePlan: ...
+
+  # Correct syntax:
+  apiGateway:
+    apiKeys: ...
+    resourcePolicy: ...
+    usagePlan: ...
+```
+
+The `schema` option on HTTP events has also been renamed to `schemas`. That change allowed us to support much more schema validation features:
+
+```yaml
+functions:
+  hello:
+    handler: hello.handler
+    events:
+      - http:
+          ...
+          request:
+            # Deprecated in v2, removed in v3:
+            schema: ...
+            # Correct syntax:
+            schemas: ...
+```
+
+Learn more about [schema changes](../deprecations.md#aws-api-gateway-schemas).
+
+When an external API Gateway is used and imported via `provider.apiGateway.restApiId`, both `provider.logs.restApi` and `provider.tracing.apiGateway` options are ignored. In v3, an error will be thrown if these options are defined. Indeed, these settings are applicable only if the API Gateway is provisioned by Serverless Framework.
+
+The CloudFormation tags defined in `provider.tags` will now be correctly applied to HTTP APIs stages ([learn more](../deprecations.md#http-api-provider-tags)).
+
+Starting with v3, AWS’ recommended behavior for API Gateway authorizers will become the default: `functions[].events[].http.authorizer.identitySource` will no longer be set to `method.request.header.Authorization` by default when caching is disabled (i.e. for authorizers of type "request" with `resultTtlInSeconds` set to "0") ([learn more](../deprecations.md#default-identitysource-for-httpauthorizer)).
```

Comment:
Is `’` intentional after `AWS`?

Reply:
It seems it depends on whether "AWS" is plural or not 😄 ("services"? or "AWS" as a whole?)  (https://www.thepunctuationguide.com/apostrophe.html)

Anyway if it looks weird that way I might as well go with the "less weird" approach if both could fit.

Reply:
I think I did not approach such notation before, but maybe I was simply ignoring it :)